### ISSUE TITLE
Using a SQL Container with a Volume

### ISSUE THREAD

**Issue by marcominerva** [dotnet/aspire-samples#44]
We need an example about how to configure SQL Server container with a Volume using Aspire. I have tried using the method `WithVolumeMount`, but with no luck.

**Comment by davidfowl** [comment]
https://github.com/dotnet/aspire/issues/974

**Comment by DamianEdwards** [comment]
Agreed this would be a good sample to have. I'm thinking of just having a default Blazor app with Identity (individual accounts auth) configured to use SQL Server, hosted via an AppHost that composes in a SQL Server container as the store.
